detection-rules/lookalike_sender_domain.yml

```diff
@@ -0,0 +1,38 @@
+name: "Lookalike sender domain"
+description: |
+  Sender's domain is a lookalike of one of your organization's domains.
+type: "rule"
+source: |
+  type.inbound
+
+  and any($org_domains,
+      // this rule can be duplicated in order to tune the threshold
+      iedit_distance(sender.email.domain.domain, .) < 2
+  )
+
+  // first-time sender
+  and (
+      (
+          sender.email.domain.root_domain in $free_email_providers
+          and sender.email.email not in $sender_emails
+      )
+      or (
+          sender.email.domain.root_domain not in $free_email_providers
+          and sender.email.domain.domain not in $sender_domains
+      )
+  )
+
+  // unsolicited
+  and (
+      (
+          sender.email.domain.root_domain in $free_email_providers
```

Comment:
looks like these two things can be merged together

Reply:
damn you're right, lots of rules to update later